config_src/infra/FMS2/MOM_interp_infra.F90

```diff
@@ -16,8 +17,8 @@ module MOM_interp_infra
 
 public :: horiz_interp_type, horiz_interp_init
 public :: time_interp_extern, init_extern_field, time_interp_external_init
-public :: get_external_field_info, axistype, get_axis_data
```

Comment:
These changes in the publicly declared interfaces from the FMS2/MOM_interp_infra module do not appear to mirror changes in the FMS1/MOM_interp_infra module.  They need to be kept identical.